src/vandermonde.jl

```diff
@@ -1,63 +1,85 @@
 export Vandermonde
+
 """
-[`Vandermonde` matrix](http://en.wikipedia.org/wiki/Vandermonde_matrix)
+    V = Vandermonde(c::AbstractVector)
```

Comment:
Not sure how it's done in this package but commonly one just uses
```suggestion
    Vandermonde(c::AbstractVector)
```